integ-test/src/test/java/org/opensearch/sql/sql/LikeQueryIT.java

```diff
@@ -3,19 +3,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 package org.opensearch.sql.sql;
 
-import org.json.JSONObject;
-import org.junit.Test;
-import org.opensearch.sql.legacy.SQLIntegTestCase;
-
-import java.io.IOException;
-
 import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_WILDCARD;
 import static org.opensearch.sql.util.MatcherUtils.rows;
 import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
 
+import java.io.IOException;
```

Comment:
Changes from other PR

Reply:
I think GJF covers order too, so later I can remove some of the non-Google Java Format tasks from the `build.gradle`